README.adoc

```diff
@@ -159,6 +159,25 @@ docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v ${PWD}/dockerfil
 If you don't want the build to redownload everytime all the sbt dependencies (it can be heavy) you can mount
 your local sbt repository as a volume by adding `-v $HOME/.ivy2/cache:/root/.ivy2/cache` to the above command.
 
+== Docker runner
+
+You can use the Dockerfile at the root to build an image that will contain this Gatling project that you can run.
+
+----
+docker build -t james-gatling-runner .
+----
+
+You can change the configuration of the Gatling project before the build, so that maybe it targets tour environment
+with the list of users you want to use for example.
+
+To launch it:
+
+----
+docker run -it --rm james-gatling-runner
+----
+
+Then you will be prompted with sbt from inside the container. You can run simulations now.
+
```

Comment:
Please document how to pass an environment variable file (the docker way) as well as customise the CSV file.